src/3rdparty/walkontable/src/table.js

```diff
@@ -635,11 +547,24 @@ class Table {
       return -4;
     }
 
+    if (row < 0) {
+      const zeroBasedHeaderLevel = (-1 * row) - 1;
```

Comment:
If we set `nestedHeaders` then we get something like that:
![image](https://user-images.githubusercontent.com/11849621/63941099-2dc97680-ca6b-11e9-8242-e3ecd6be27c9.png)

and to get `TH` for header ` N ` we have to call `getCell(-4, 0)` but in my opinion it should be `getCell(-1, 0)`. What do you think?

Reply:
BTW. What error code I should get if the cell is in a row before viewport and in column after viewport?
(good catch @wojciechczerniak! :smile::+1:)

Reply:
> BTW. What error code I should get if the cell is in a row before viewport and in column after viewport?

The method `getCell` only returns one error code. It validates the input parameters in order in which they are given. Thus, row is validated first and you will get the error code for the row before viewport.

Reply:
> and to get TH for header N we have to call getCell(-4, 0) but in my opinion it should be getCell(-1, 0). What do you think?

I guess negative numbers always will be a bit confusing, whatever way we go.

The current order (`-4, 0` returns `N`) comes from the fact, in the Nested Headers concept of "levels", level 0 corresponds to the row that has `A`, level 1 to `D`, level 2 to `H`, and level 3 to `N`.

Thus, to retrieve nested header level `n` from `getCell`, you need to pass `(n + 1) * -1` as the row index.

Of course, I can change that to be `(levelsCount - n) * -1` but are you sure that would be clearer?

For now, I have not changed the behavior of `getCell`, but added a proper test in 541bcb36295e7565dc1d19fa5043c7fa212d90a3.

@swistach, @jansiegel, @wojciechczerniak WDYT?

Reply:
I think `N` should be obtained with `getCell(-1, 0)` too. The concept of Nested Headers' levels is connected the the fact that they're nested, so higher "level" = nested deeper, but I don't think it translates too well for `getCell` and might potentially be confusing.

Reply:
No problem, I have just pushed an implementation that reverses the behavior: https://github.com/handsontable/handsontable/pull/6200/commits/07aef23282374b4a3e630d6ca1968b6f51aa047a